### ISSUE TITLE
netCDF input: TypeError about datetime raised

### ISSUE THREAD

**Issue by qAp** [xray/xray#347]
Hi,
I'm having a problem opening and importing data from a netCDF file produced by some atmosphere model:
```
In [38]: ls
atm_in                               camrun.cpl.r.0095-08-01-00600.nc
bld/                                 camrun.cpl.r.0095-08-04-21000.nc
cam.log                              ccsm_timing
camrun.cam.h0.0095-07-18-19800.nc    chem_mech.in
camrun.cam.r.0095-08-01-00600.nc     drv_flds_in
camrun.cam.r.0095-08-04-21000.nc     drv_in
camrun.cam.rs.0095-08-01-00600.nc    ice_in
camrun.cam.rs.0095-08-04-21000.nc    lnd_in
camrun.clm2.h0.0095-07.nc            ocn_in
camrun.clm2.r.0095-08-01-00600.nc    rof_in
camrun.clm2.r.0095-08-04-21000.nc    rpointer.atm
camrun.clm2.rh0.0095-08-01-00600.nc  rpointer.drv
camrun.clm2.rh0.0095-08-04-21000.nc  rpointer.lnd

In [39]: h0rrtmg = xray.open_dataset("camrun.cam.h0.0095-07-18-19800.nc")

In [40]: h0rrtmg
Out[40]: <repr(<xray.core.dataset.Dataset at 0x2aab65ffd7b8>) failed: TypeError: Cannot cast datetime.date object from metadata [D] to [ns] according to the rule 'same_kind'>
```

I can open it using the netCDF4 module
```
In [51]: h0rrtmg = netCDF4.Dataset('camrun.cam.h0.0095-07-18-19800.nc', 'r')

In [52]: h0rrtmg.variables['time']
Out[52]: <netCDF4.Variable at 0x2aab6607fdc8>

In [53]: h0rrtmg.variables['time'].units
Out[53]: 'days since 0095-07-18 05:30:00'

In [54]: h0rrtmg.variables['time'].dtype
Out[54]: dtype('float64')

In [55]: h0rrtmg.variables['time'][:5]
Out[55]: array([ 0.        ,  0.01388889,  0.02777778,  0.04166667,  0.05555556])
```
What is happening here?  Is there something that I ought to change in the .nc file before using xray, or should something change in xray so that it can read this .nc file?

The .nc file can be found here: https://drive.google.com/file/d/0B68-5koyGA7IOG9ydVhkbVh1SUU/view?usp=sharing

Thanks

**Comment by shoyer** [comment]
Hmm. This is different from the usual datetime decoding issue I've noticed. I'll investigate.

For now, use `decode_times=False` when calling `open_dataset`.

**Comment by shoyer** [comment]
PR #350 fixes this. It will be included in the next xray release (v0.4), which will come out in a day or day.
